omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -86,10 +86,11 @@
     location ~ "^/organizations/([^/]+)/data-collector$" {
       set $request_org $1;
       access_by_lua_block { validator.validate("POST") }
+      proxy_set_header Host $proxy_host;
       proxy_set_header x-data-collector-token $data_collector_token;
       proxy_set_header x-data-collector-auth "version=1.0";
       rewrite ^ <%= URI.parse(node['private_chef']['data_collector']['root_url']).path %> break;
-      proxy_pass <%= URI.parse(node['private_chef']['data_collector']['root_url']).scheme %>://data-collector;
+      proxy_pass <%= URI.parse(node['private_chef']['data_collector']['root_url']).scheme %>://<%= URI.parse(node['private_chef']['data_collector']['root_url']).host %>:<%= URI.parse(node['private_chef']['data_collector']['root_url']).port %>;
```

Comment:
With this change, do we still need to keep the `data-collector` location defined above?